src/main/resources/org/sagebionetworks/web/client/view/TeamViewImpl.ui.xml

```diff
@@ -29,10 +29,10 @@
 							iconStyles="lightGreyText font-size-18 displayInlineBlock"
 							addStyleNames="margin-top-15 margin-bottom-10" placement="LEFT" />
 						<b:DropDown ui:field="dashboardDropdown"
-							addStyleNames="imageButton SRC-primary-background-color-hover displayInlineBlock"
```

Comment:
Caused all menu items to have the hover style, move this class to the menu item.